sync/commit.py

```diff
@@ -160,7 +160,11 @@ def is_empty(self, prefix=None):
             args = ("--", prefix)
         else:
             args = ()
-        return self.repo.git.show(self.sha1, format="", patch=True, *args).strip() == ""
+        return self.repo.git.show(self.sha1,
+                                  format="",
+                                  patch=True,
+                                  stdout_as_string=False,
```

Comment:
`False` is the default so you can keep the line untouched.

Reply:
No, `True` is the default, that's the bug that's being fixed here

Reply:
So what does this line do then? https://github.com/mozilla/wpt-sync/blob/c50d5d75743355c403bead18391a8f8040b85888/sync/commit.py#L187

Reply:
That code isn't being called in this case.

Reply:
So this shows my missing knowledge of this code as pointed out 27 days ago. Given that I don't have the time to dive into all this, I trust you and the passing new tests. Lets get it shipped.